contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -566,13 +570,17 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
     function _sendTeleporterMessage(
         TeleporterMessageInput memory messageInput,
         TeleporterMessageReceipt[] memory receipts
-    ) private returns (uint256) {
+    ) private returns (bytes32) {
+        // If the blockchain ID has yet to be initialized, do so now.
+        bytes32 blockchainID_ = _initializeBlockchainID();
+        require(blockchainID_ != bytes32(0), "TeleporterMessenger: blockchainID not set");
```

Comment:
This check seems unnecessary. We already check against the `bytes32(0)` case inside the function, so this should never occur.

If we want to keep this as a redundant check, can we also add this `require` in the same place in `receiveCrossChainMessage`?

Reply:
I think we should remove the redundant check